src/Uno.Toolkit.UI/Behaviors/FlipViewExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public static class FlipViewExtensions
+{
+	#region DependencyProperty: Next
+	public static DependencyProperty NextProperty { get; } =
+	DependencyProperty.RegisterAttached("Next", typeof(FlipView), typeof(FlipViewExtensions), new PropertyMetadata(null, OnNextChanged));
+
+	public static void SetNext(Button element, FlipView value) => element.SetValue(NextProperty, value);
+
+	public static FlipView GetNext(Button element) => (FlipView)element.GetValue(NextProperty);
+
+	#endregion
+
+	#region DependencyProperty: Previous
+	public static DependencyProperty PreviousProperty { get; } =
+	DependencyProperty.RegisterAttached("Previous", typeof(FlipView), typeof(FlipViewExtensions), new PropertyMetadata(null, OnPreviousChanged));
+
+	public static void SetPrevious(Button element, FlipView value) => element.SetValue(PreviousProperty, value);
+
+	public static FlipView GetPrevious(Button element) => (FlipView)element.GetValue(PreviousProperty);
+	#endregion
+
+	static void OnNextChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+	{
+		var btn = (ButtonBase)d;
+
+		if (e.NewValue is null)
+		{
+			btn.Click -= OnBtnClick;
+			return;
+		}
+
+		var flipView = (FlipView)e.NewValue;
+
+		btn.Click -= OnBtnClick;
+		btn.Click += OnBtnClick;
```

Comment:
This is going to create a memory leak (the extension will create a hard link between the button and the flipview. @kazo0 and @Xiaoy312 have some recent examples of how to handle this.

Reply:
@kazo0 We'll likely need to include the [memory leaks runtime test](https://github.com/unoplatform/uno/blob/56d3ba98ee6cab055dc8d7c4941ce4494e20e946/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_And_Leak.cs) found in Uno to catch issues like these.

Reply:
@jeromelaban I can try to test it again using the sample project. While ago I isolated an old implementation [here](https://github.com/pictos/UnoSpecs/blob/pj/FlipView-Memory-Leak/UnoSpecs/UnoSpecs.Shared/FliperAttachedProperty.cs) and during my tests the page ways get collected.
I made all the implementation on Toolkit `static`, to make sure the `delegate` isn't capturing anything.

But if you still want, I can implement a weak subcriber, like [this one](https://github.com/maxkoshevoi/XamarinCommunityToolkit/blob/735f0100e45f1a22116bec3b32248b69f03b13f6/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/INotifyPropertyChangedExtension.shared.cs)

Reply:
Changed the code to use the same approach used on FlipViewExtensions